nc2zarr/res/config-template.yml

```
@@ -87,18 +87,28 @@ process:
 
   # (Re)chunk variable dimensions
   rechunk:
-    # Set selected dimensions of all variables to chunk_size.
+    # For all variables:
     "*":
+      # Set chunk size for <dim_name> to <chunk_size>.
       <dim_name>: <chunk_size>
-      <dim_name>: <chunk_size>
-    # Set selected dimensions of individual variables to chunk_size.
+      # Set chunk size for <dim_name> to dimension size.
+      <dim_name>: null
```

```suggestion
      <dim_name>: "dimension"
```

See suggestion at line 79 of `processor.py`. (And similarly for other `null` `dim_name`s below.)

---

Not now.